components/sidebar/sidebar_channel_button_or_link/sidebar_channel_button_or_link.jsx

```diff
@@ -44,6 +45,14 @@ export default class SidebarChannelButtonOrLink extends React.PureComponent {
 
     render = () => {
         let badge = null;
+        let usernamePlaceHolder = this.props.displayName;
+        if (!usernamePlaceHolder) {
+            usernamePlaceHolder = (
+                <span className='loading-user'>
+                    <LoadingScreen style={{padding: '0px'}}/>
+                </span>
+            );
```

Comment:
should we try and get the user here or is that handled somewhere else?

Reply:
It is handled just happens for a brief time before user is loaded. That being said there is no way for us to know if we made a request or request status of it. 

We should change sometime the behavior of user request status to take userID into account so we know if a particular user request is initiated/failed